CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+
+## [3.11.6] - 2023-10-24
+### Fixed
+- Switch to ExecutorService for `MetastoreMappingImpl.isAvailable()` calls to prevent prevent default commonPool usage and threads waiting on each other.
```

Comment:
```suggestion
- Switch to ExecutorService instead of the default `ForkJoinPool` for `MetastoreMappingImpl.isAvailable()` calls. Using `ForkJoinPool` causes threads to wait on each other.
```